main.js

```diff
@@ -0,0 +1,49 @@
+/* Object constructor */
+function Book(name, author) {
+  this.name = name;
+  this.author = author;
+}
+
+/* Create booksArray and set it to empty if localStorage is null */
+const booksArray = JSON.parse(localStorage.getItem('books')) || [];
+
+/* Selector */
+const bookList = document.querySelector('.book-list');
+const bookName = document.querySelector('.book-name');
+const author = document.querySelector('.author');
+const form = document.querySelector('form');
+
+/* Functions */
+function displayBooks() {
+  const books = JSON.parse(localStorage.getItem('books'));
+  bookList.innerHTML = books.map((book, i) => `
+    <li class="book">
+      <h2 class="book-title">${book.name}</h2>
+      <p class="book-author">${book.author}</p>
+      <button class="remove-book" data-index=${i}>Remove</button>
+    </li>
+    <hr style="width: 500px; margin-left: 0;">
+    `).join('');
+}
```

Comment:
- [ ] Your add and remove book functions are working well, and the `displayBooks` too. But if the user reloads the page, the book list is not displayed because this function is not being called when the page loads. Please implement functionality to display the books when the page reloads so the localStorage data can be read and properly used.